examples/input_processing.rs

```diff
@@ -0,0 +1,70 @@
+use bevy::prelude::*;
+use leafwing_input_manager::prelude::*;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_plugins(InputManagerPlugin::<Action>::default())
+        .add_systems(Startup, spawn_player)
+        .add_systems(Update, check_data)
+        .run();
+}
+
+#[derive(Actionlike, PartialEq, Eq, Clone, Copy, Hash, Debug, Reflect)]
+enum Action {
+    Move,
+    LookAround,
+}
+
+#[derive(Component)]
+struct Player;
+
+fn spawn_player(mut commands: Commands) {
+    let mut input_map = InputMap::default();
+    input_map
+        .insert(
+            Action::Move,
+            VirtualDPad::wasd()
+                // You can add a processor to handle axis-like user inputs by using the `with_processor`.
+                //
+                // This processor is a circular deadzone that normalizes input values
+                // by clamping their magnitude to a maximum of 1.0,
+                // excluding those with a magnitude less than 0.1,
+                // and scaling other values linearly in between.
+                .with_processor(CircleDeadzone::default())
+                // Followed by appending another processor for the next processing step.
+                .with_processor(AxisSensitivity(2.0).extend_dual()),
```

Comment:
`extend_dual` feels like boilerplate to me. Can we remove it with `impl Into` style tricks?

Reply:
Since the implementations generated by define_dual_axis_processor are enum variants (All, Separate(x, y), OnlyX, OnlyY), I feel like repeating code such as `DualAxisSensitivity::All(AxisSensitivity(2.0))` would be more cumbersome